pages/post/detail/[post_id].vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div>
+
+    <UContainer v-if="post">
```

Comment:
empty object or Object like data are considered as true, so this check does not make any sense here.

Reply:
Although if you’re not rending anything till the request is resolved you can show a loader